### DISCUSSION TITLE
Fix the broken "bright" color misuse in terminal output

### DISCUSSION THREAD

**Discussion by webknjaz** [ansible/ansible-lint#834] [Ideas]
This is problematic because it'll only work in terminals that support misusing the bold setting for bright. Standard-compliant terminals don't work like that.

It's roughly summarized here: https://github.com/kovidgoyal/kitty/issues/197#issuecomment-348062665. And while for red it's semi-okay, for things like grey it's a disaster because it looks the same as the background == looks like empty lines.

_Originally posted by @webknjaz in https://github.com/ansible/ansible-lint/pull/833/threads/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkNDQwNzcyODkw/unresolve_

**Comment by ssbarnea** [comment]
I am ok to "fix" it once ansible fixes it first, see below result of search on ansible devel branch:

```
$ rg bright                                                                                                                                                               README.rst
93:.. |Docs badge| image:: https://img.shields.io/badge/docs-latest-brightgreen.svg
97:.. |Chat badge| image:: https://img.shields.io/badge/chat-IRC-brightgreen.svg
105:.. |License| image:: https://img.shields.io/badge/license-GPL%20v3.0-brightgreen.svg

lib/ansible/config/base.yml
312:  default: bright red
326:  default: bright purple

examples/ansible.cfg
472:#warn = bright purple

lib/ansible/galaxy/login.py
55:        display.display(u'\n\n' + "We need your " + stringc("GitHub login", 'bright cyan') +
57:        display.display("This information will " + stringc("not be sent to Galaxy", 'bright cyan') +

lib/ansible/plugins/callback/dense.py
76:    COLOR_WARN = 'bright purple'
81:    COLOR_UNREACHABLE = 'bright red'

lib/ansible/utils/color.py
58:    'black': u'0;30', 'bright gray': u'0;37',
60:    'green': u'0;32', 'bright blue': u'1;34',
61:    'cyan': u'0;36', 'bright green': u'1;32',
62:    'red': u'0;31', 'bright cyan': u'1;36',
63:    'purple': u'0;35', 'bright red': u'1;31',
64:    'yellow': u'0;33', 'bright purple': u'1;35',
65:    'dark gray': u'1;30', 'bright yellow': u'1;33',
66:    'magenta': u'0;35', 'bright magenta': u'1;35',

test/units/utils/display/test_warning.py
24:    mocker.patch('ansible.utils.color.parsecolor', return_value=u'1;35')  # value for 'bright purple'
```

**Comment by webknjaz** [comment]
Ansible has a lot of legacy you cannot seriously suggest supporting broken things because somebody else also makes mistakes.

P.S. I think I may have some year-old experiments with fixing the color support in ansible/ansible somewhere locally but it's incomplete. Still fixing it here is easier because it's just one place that's broken.
